tests/MongoDbTest.php

```diff
@@ -42,19 +42,55 @@ public function it_can_generate_a_dump_command_with_compression_enabled()
             ->getDumpCommand('dbname.gz');
 
         $this->assertSame('\'mongodump\' --db dbname'
-            .' --archive --host localhost --port 27017 | gzip > "dbname.gz"', $dumpCommand);
+            .' --archive --host localhost --port 27017 | gzip -9 > "dbname.gz"', $dumpCommand);
```

Comment:
All of the current tests should keep working without any modification. If you change them, your code changes are probably breaking and can't be released without creating a new major version.

Reply:
Sorry, It wasn't the intention.